firedrake/slate/slac/kernel_builder.py

```diff
@@ -161,21 +168,16 @@ def _setup(self):
             for split_kernel in cxt_kernel.tsfc_kernels:
                 indices = split_kernel.indices
                 kinfo = split_kernel.kinfo
-
-                # TODO: Implement subdomains for Slate tensors
-                if kinfo.subdomain_id != "otherwise":
-                    raise NotImplementedError("Subdomains not implemented.")
+                kint_type = kinfo.integral_type
 
                 args = [c for i in kinfo.coefficient_map
                         for c in self.coefficient(local_coefficients[i])]
 
                 if kinfo.oriented:
                     args.insert(0, self.cell_orientations_sym)
 
-                if kinfo.integral_type in ["interior_facet",
-                                           "exterior_facet",
-                                           "interior_facet_vert",
-                                           "exterior_facet_vert"]:
+                if kint_type in ["interior_facet", "exterior_facet",
+                                 "interior_facet_vert", "exterior_facet_vert"]:
```

Comment:
FWIW, it's best not to introduce unnecessary whitespace changes in functionality patches.  I was trying to understand what changed here (the answer was nothing).